src/test/modelmanager_test.cpp

```diff
@@ -321,15 +321,6 @@ TEST(ModelManager, ConfigReloadingShouldAddNewModel) {
     modelMock.reset();
 }
 
-TEST(ModelManager, ConfigReloadingWithWrongInputName) {
```

Comment:
Why this test is removed?

Reply:
Because we no longer maintain status throughout the function - failures are non blocking. They are reported in spdlog.

Reply:
I don't see any code changing reloadModelWithVersions behavior. Why do we change the test now?